css/style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-areas: 'main' 'promo' 'footer';
+}
+
+.flex-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.flex-col {
+  display: flex;
+  flex-direction: column;
+}
+
+/* background, navbar and titles starts here */
+
+.main {
+  grid-area: main;
+  height: 538px;
+  background: url(../images/main_hero.jpg) 50% 100% no-repeat;
+  background-size: cover;
+  align-items: center;
+}
+
+.main .nav-bar {
+  margin: 90px 0 30px 0;
+  justify-content: center;
+  align-items: center;
+  background-image:
+    linear-gradient(
+      rgb(116, 116, 116) 30%,
+      rgb(111, 111, 111) 49%,
+      rgb(90, 90, 90) 50%,
+      rgb(116, 116, 116) 90%
+    );
+  border-radius: 4px;
+  color: white;
+}
+
+.nav-bar > span {
+  cursor: pointer;
+  padding: 9px 27px;
+  border-right: 1px solid #484848;
+}
+
+.main .nav-bar span:hover {
+  background: rgb(70, 70, 70);
+}
+
+.footer div {
+  font-size: 11px;
+  line-height: 18px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.footer div p a {
+  text-decoration: none;
+}
+
+.main .nav-bar span a {
+  color: white;
+  font-family: lucida sans, lucida sans regular, lucida grande, lucida sans unicode, Geneva, Verdana, sans-serif;
+  text-decoration: none;
+}
+
+.footer div p a:hover {
+  text-decoration: underline;
+}
+
+.main .nav-bar span.mag-span {
+  position: relative;
+}
+
+.main .nav-bar span span.mag {
+  position: absolute;
+}
+
+.main .nav-bar span input {
+  width: 80px;
+  border-radius: 25px;
+  max-height: 13px;
+  padding-left: 18px;
+  background: #747474;
+  border: 1px #616161 solid;
+}
+
+.main .nav-bar span input:hover {
+  width: 80px;
+  border-radius: 25px;
+  max-height: 13px;
+  padding-left: 18px;
+  background: white;
+  border: 1px #616161 solid;
+}
+
+.main .nav-bar span img {
+  max-height: 18px;
+}
+
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<img>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.main .nav-bar span img {

/* after */
.imgtag-class {
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_